pkg/dataMover/data_mover.go

```diff
@@ -65,9 +68,9 @@ func (this *DataMover) CopyToRepo(peID astrolabe.ProtectedEntityID) (astrolabe.P
 		return astrolabe.ProtectedEntityID{}, err
 	}
 
-	this.Infof("Ready to call s3 PETM copy API")
+	this.Debugf("Ready to call s3 PETM copy API")
 	s3PE, err := this.s3PETM.Copy(ctx, updatedPE, astrolabe.AllocateNewObject)
-	this.Infof("Return from the call of s3 PETM copy API")
+	this.Debugf("Return from the call of s3 PETM copy API")
```

Comment:
Please add the PEID we're copying from.  This helps if log messages get interleaved so we can know which ones are completing

Reply:
Same above.